docs/raspberry-pi_install.md

```diff
@@ -118,3 +118,36 @@ bell after pressing button to confirm SUSI has started listening. Ask your query
 
 If you still face any errors in the setup, please provide a screenshot or logs of errors being encountered.
 This would help rectify the issue.
+
+
+# Installation on Raspbian Lite
+- Tested on
+- Raspberry Pi 3 with Raspbian Strech Lite
+
+You will need to have access to Raspberry Pi Screen and terminal for the initial setup.
+You may start from ssh later.
+
+## STEPS
+- First configure wifi on your pi
+    - Open the following file ```` sudo nano /etc/wpa_supplicant/wpa_supplicant.conf ````
```

Comment:
You can include the step where user can enable SSH, configure wifi and keyboard from `sudo raspi-config`
https://www.raspberrypi.org/documentation/configuration/raspi-config.md